### PR TITLE
Show warning console if version.js file is missing

### PR DESCRIPTION
**Pull request by sujono91**
## Motivation

As discussed in #369, we have to give better error message if `version.js` file is missing in `pages/en` directory.

The solution was to add warning console if someone ran the version script before creating the `versions.js` page

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/Docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

Warning console example if `version.js` file is not found
screen shot 2018-05-20 at 10 33 46 am

Success example if `version.js` file is found
image

Run `yarn test` and it pass all unit test

## Related PRs

No

### CODE REVIEW

**Comment by endiliey** [comment]
Some suggestion. Instead of just a warning when `version.js` is missing, will it be better to prompt user if they want to create a default `version.js` ?  This is more from user experience (UX) perspective.  Showing `https://docusaurus.io/docs/en/versioning.html` as a reference guide might be useful as well. WDYT?

**Comment by sujono91** [comment]
@endiliey i agree with u, but can u give me the example of how to create a default `version.js`?

@JoelMarcey @yangshun do u've any comments about this?

**Comment by endiliey** [comment]
Refer to https://docusaurus.io/docs/en/versioning.html, running `yarn examples versions` create an example `pages/en/versions.js`

`yarn examples versions` is actually just running `node ../lib/copy-examples.js versions` https://github.com/facebook/Docusaurus/blob/aae106c018667a3787726f7744ce14ccb2b68ef1/lib/copy-examples.js#L97-L122

**Comment by supasate** [comment]
Showing `https://docusaurus.io/docs/en/versioning.html` as a reference guide is useful and nice to be included in this PR.

For prompting and creating a default `version.js`, should we separate into a new issue/PR? So, this PR only focuses on a better error message and the new PR focuses only the new functionality to create a default `versions.js` if not exists.

**Comment by sujono91** [comment]
@endiliey @supasate @JoelMarcey @yangshun 
My Suggestion:
```
No version.js file founded!
You should create your version.js file in pages/en directory.
Please refer to https://docusaurus.io/docs/en/versioning.html
```

**Comment by JoelMarcey** [comment]
@sujono91 That definitely seems reasonable to me. Much better than what we have now -- which is nothing 😄. We can always do any minor tweaking later, if we want.

File: lib/version.js [hunk 1]
```diff
@@ -33,6 +36,15 @@ program
   })
   .parse(process.argv);
 
+if (!hasVersionFile) {
+  console.error(
+    `${chalk.yellow(
+      'No version.js file founded!'
```
Comment (yangshun) [inline lib/version.js]: Should it be `versions.js` or `version.js`? The code checks for `versions.js` but the error message suggests `version.js`.

Grammatical nit - The error message should be "No versions.js file found" not founded.

Could you also shift the `const hasVersionFile` declaration to just above the check here? We would like to co-locate the variables to where they are being referenced, especially when this file is only being referenced once.

File: lib/version.js [hunk 2]
```diff
@@ -33,6 +36,15 @@ program
...
+    )}\nPlease create your version.js file in pages/en directory.`
```
Comment (yangshun) [inline lib/version.js]: Sync filename with above.

**Comment by sujono91** [comment]
@yangshun @JoelMarcey updated

**Comment by sujono91** [comment]
Updated @yangshun @JoelMarcey

**Pull request merged**